NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1Page.xaml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<ContentPage x:Class="NUITizenGallery.VectorGraphicsCanvasViewTest1Page"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+  xmlns:base="clr-namespace:Tizen.NUI.BaseComponents.VectorGraphics;assembly=Tizen.NUI"

```

Comment:
vector would be better than base... but not important thing..

Reply:
It's just taken from other code.
I'll fix it when I add the next test.